app/elements/challenge/challenge-modal.html

```diff
@@ -69,7 +70,17 @@
 			onChallengeResponse: function (res) {
 				this.fire('challengeCreated');
 				page.redirect('/challenges/' + res.detail.response.Key);
+			},
+			_computeDuration: function(){
+				var duration = 0;
+				for (var i = 0; i < this.selectedTasks.length; i++){
+					duration += this.selectedTasks[i].Duration;
+				}
+				this.Duration = duration / 6 / 1e10;
```

Comment:
Because the Challenge does not exist yet. This is the form for creating a Challenge. And the challenge duration field will be initialized with the sum of durations from the selected tasks. It can still be changed, of course. 
- In the future here, we need to have a validity check for the duration of the challenge(not lower than the longest task for paralel and not lower than the sum of tasks for sequential)